src/components/MapBox.tsx

```diff
@@ -67,17 +65,6 @@ const MapSettings: React.FC<{
     logger.debug(`zoom: ${zoom}`);
   }, [zoom]);
 
-  useEffect(() => {
-    const h3 = geoToH3(center[0], center[1], utils.constants.DefaultH3Resolution);
```

Comment:
What is `h3` ?

Reply:
that was hexagons on map (not required for new implementation) https://h3geo.org/docs/